config.lua

```diff
@@ -441,6 +441,42 @@ lspconfig.pylsp = add_lsp {
   command = { 'pylsp' },
   verbose = false
 }
+---# Ruby LSP
```

Comment:
Could you add a newline above this line?

Reply:
sure, done :)